### DISCUSSION TITLE
Faikin on ESP32 NodeMCU Development Board

### DISCUSSION THREAD

**Discussion by openKimchi** [revk/xESP32-Faikout#160] [General]
Hi,
flashed an ESP32 NodeMCU Development Board and it works so far(not yet connencted to air conditioner) How can I connect it to my Perfera cause there is no connection for 12 V on this ESP Board?

**Comment by revk** [comment]
I believe some Daikin have an X35 connector, but otherwise, no idea.

**Reply by openKimchi** [reply]
OK, thx! My Perfera FTXM42R does not seem to have an X35 connector. So I better wait for the original Faikin Board I ordered from Amazon a few days ago. Lucky me they ship to Germany. I also ordered a Daikin EKRS21 cable. Hope that will be enough to get it work.

**Reply by revk** [reply]
Very possibly. The Faikin can work from about 4V to 35V if needed.
